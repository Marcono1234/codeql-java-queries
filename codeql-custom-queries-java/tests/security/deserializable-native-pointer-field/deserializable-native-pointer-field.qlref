./security/deserializable-native-pointer-field.ql
