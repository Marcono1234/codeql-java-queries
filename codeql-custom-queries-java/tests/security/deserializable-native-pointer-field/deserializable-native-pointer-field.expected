| Test.java:19:22:19:33 | otherAddress | Field seems to represent native pointer, but can be deserialized |
| Test.java:20:22:20:28 | pointer | Field seems to represent native pointer, but can be deserialized |
